src/components/Tokens/TokenDetails/PriceChart.tsx

```diff
@@ -27,7 +27,7 @@ import {
 import LineChart from '../../Charts/LineChart'
 
 // TODO: This should be combined with the logic in TimeSelector.
-const TIME_DISPLAYS: [TimePeriod, string][] = [
+export const TIME_DISPLAYS: [TimePeriod, string][] = [
```

Comment:
this really shouldn't be an array?

Reply:
there should `displays: Record<TimePeriod, string`, `orderedDisplays: TimePeriod[]`, and `TimePeriod`